bioblend/galaxy/objects/client.py

```diff
@@ -326,6 +327,74 @@ def delete(self, id_=None, name=None):
                 self._error('delete_workflow: unexpected reply: %r' % res)
 
 
+class ObjInvocationClient(ObjClient):
+    """
+    Interacts with Galaxy Invocations.
+    """
+    def get(self, id_) -> wrappers.Invocation:
+        """
+        Get an invocation by ID.
+
+        :rtype: Invocation
+        :param: invocation object
+        """
+        inv_dict = self.gi.invocations.show_invocation(id_)
+        return wrappers.Invocation(inv_dict, self.obj_gi)
+
+    def get_previews(self) -> List[wrappers.InvocationPreview]:
+        """
+        Get previews of all invocations.
+
+        :rtype: list of InvocationPreview
+        :param: previews of invocations
+        """
+        inv_list = self.gi.invocations.get_invocations()
+        return [wrappers.InvocationPreview(inv_dict, self.obj_gi) for inv_dict in inv_list]
+
+    def list(
+        self,
+        workflow_id=None,
+        history_id=None,
+        user_id=None,
```

Comment:
In general BioBlend.objects methods shouldn't be passed ids (apart from `get()`), so I think these parameters should be of type `wrappers.Workflow` and `wrappers.History`, dropping `user_id` for now (the objects interface is not tailored towards admins any way).